arch/arm64/boot/dts/qcom/sm7325-nothing-spacewar.dts

```diff
@@ -1085,6 +1100,43 @@
 	status = "okay";
 };
 
+&sound {
+	compatible = "qcom,sm7325-sndcard",
+		     "qcom,sm8250-sndcard";
```

Comment:
On FP5 we use `"qcom,qcm6490-idp-sndcard"` as second compatible, would fit here better also. That's from sc8280xp driver iirc